py2hwsw/lib/hardware/registers/regarray/iob_regarray_sp/iob_regarray_sp.py

```diff
@@ -5,26 +5,139 @@
 
 def setup(py_params_dict):
     attributes_dict = {
-        "generate_hw": False,
+        "generate_hw": True,
+        "confs": [
+            {
+                "name": "ADDR_W",
+                "type": "P",
+                "val": 2,
+                "min": "NA",
+                "max": "NA",
+                "descr": "Address width in bits",
+            },
+            {
+                "name": "DATA_W",
+                "type": "P",
+                "val": 21,
+                "min": "NA",
+                "max": "NA",
+                "descr": "Width of the data in bits",
+            },
+        ],
         "ports": [
             {
                 "name": "clk_en_rst_s",
                 "signals": {
                     "type": "iob_clk",
+                    "params": "c_a_r_e",
```

Comment:
```suggestion
                    "params": "c_a_r",
```
I don't think the `en_i` port signal is used in this module.

Reply:
why use care instead o car?

Reply:
Yes, it should be c_a_r. This has already been resolved. I hid this comment as resolved: https://github.com/IObundle/py2hwsw/pull/380#discussion_r2258551726

Reply:
oops, old commit